ui/components/app/account-menu/account-menu.component.js

```diff
@@ -320,10 +324,10 @@ export default class AccountMenu extends Component {
     }
 
     let supportText = t('support');
-    let supportLink = 'https://support.metamask.io';
+    let supportLink = SUPPORT_LINK;
     if (isBeta()) {
```

Comment:
This actually might be exactly what we need for Flask as well - another explicit request for feedback.

Maybe we can move this into a code fence, and use `needHelpSubmitTicket` for Beta, but something more general for Flask?

Reply:
Fixed in https://github.com/MetaMask/metamask-extension/pull/13226/commits/6286bfffacb905c16beb9944d1aab091fcbf4b82

Reply:
So that commit did this for the Home page footer, but this line is for the account menu. It seems that that hasn't changed yet.

It seems reasonable to make the equivalent change here in the account menu too maybe?

Reply:
Yeah, that makes sense. Change made here: ae63fd6